ruby/ql/test/library-tests/frameworks/active_record/ActiveRecord.expected

```diff
@@ -105,46 +108,47 @@ activeRecordInstances
 | associations.rb:53:1:53:34 | call to find |
 activeRecordSqlExecutionRanges
 | ActiveRecord.rb:9:33:9:67 | "name='#{...}' and pass='#{...}'" |
-| ActiveRecord.rb:19:16:19:24 | condition |
-| ActiveRecord.rb:28:30:28:44 | ...[...] |
-| ActiveRecord.rb:29:20:29:42 | "id = '#{...}'" |
-| ActiveRecord.rb:30:21:30:45 | call to [] |
-| ActiveRecord.rb:31:16:31:21 | <<-SQL |
-| ActiveRecord.rb:34:20:34:47 | "user.id = '#{...}'" |
-| ActiveRecord.rb:46:20:46:32 | ... + ... |
-| ActiveRecord.rb:52:16:52:28 | "name #{...}" |
-| ActiveRecord.rb:56:20:56:39 | "username = #{...}" |
-| ActiveRecord.rb:68:21:68:44 | ...[...] |
-| ActiveRecord.rb:106:27:106:76 | "this is an unsafe annotation:..." |
+| ActiveRecord.rb:17:24:17:24 | q |
```

Comment:
Hard to spot, but this is the the argument to `connection.execute` in the new test case.